build.gradle

```diff
@@ -74,6 +74,13 @@ subprojects { project ->
       testRuntimeOnly("org.junit.vintage:junit-vintage-engine")
       testRuntimeOnly("org.objenesis:objenesis")
     }
+
+    configurations.all {
```

Comment:
Be good to get this commented, with links to kork as NOMINALLY these should be kept SOMEWHAT in sync, and/or figure out how to do verion bumps.  Specificlly linked to kork's GCS secrets support.

Hoping monorepo *cough cough* will help!

Reply:
I hope, this would work for now.